src/components/AssetHeader/AssetChart.tsx

```diff
@@ -59,7 +59,7 @@ export const AssetChart = ({ accountId, assetId, isLoaded }: AssetChartProps) =>
   } = useLocaleFormatter({ fiatType: 'USD' })
   const [percentChange, setPercentChange] = useState(0)
   const alertIconColor = useColorModeValue('blue.500', 'blue.200')
-  const [timeframe, setTimeframe] = useState(HistoryTimeframe.MONTH)
+  const [timeframe, setTimeframe] = useState(DEFAULT_HISTORY_TIMEFRAME)
```

Comment:
use a centralized source of truth for the default allows us to prefetch this data